include/COLA.hh

```diff
@@ -0,0 +1,225 @@
+#ifndef COLA_COLA_HH
+#define COLA_COLA_HH
+
+#include <map>
+#include <memory>
+#include <vector>
+#include <cmath>
+
+namespace cola {
+
+    /* ----------------------- DATA ----------------------- */
+
+    //what happened to particle after generation
+    enum class ParticleClass: char {
+        produced,
+        elasticA,
+        elasticB,
+        nonelasticA,
+        nonelasticB,
+        spectatorA,
+        spectatorB
+    };
+
+    //initial nuclei nad kinematics, diagnostics information
+    struct EventIniState{
```

Comment:
Do we neglect the nucleons coordinates vector?

Reply:
No, it is in another structure. The inistate has nothing to do with produced nucleons, their info is in Particle vecctor, stored in the aggregated data structure.

COLA.hh has no new code at all, it is just composed from all the files we previously had